src/components/common/Navbars/NavBar.js

```diff
@@ -100,20 +102,20 @@ function NavBar(props) {
           )}
           <NavLink to="/login">
             <button
-              className={`navbar__btn navbar__login ${
-                localStorage.getItem('okta-token-storage') ? 'navbar__hide' : ''
-              }`}
+            // className={`navbar__btn navbar__login ${
+            // localStorage.getItem({}) ? 'navbar__hide' : ''
+            // }`}
             >
               LOGIN
             </button>
           </NavLink>
-          {localStorage.getItem('okta-token-storage') && (
-            <NavLink to="/parent/family">
-              <div className="navbar__profile">
-                <ProfileIcon style={{ color: 'black', fontSize: 30 }} />
-              </div>
-            </NavLink>
-          )}
+          {/* {localStorage.getItem({}) && ( */}
+          <NavLink to="/parent/family">
```

Comment:
Excellent job mapping out the navlinks for the new Auth0 object when it is created

Reply:
Thank you